package.json

```diff
@@ -1,43 +1,46 @@
 {
-  "private": true,
   "scripts": {
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
     "lint:fix": "eslint . --fix",
+    "ts-check": "tsc --noEmit",
     "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,css,scss,md}\"",
     "format:check": "prettier --check \"**/*.{js,jsx,ts,tsx,json,css,scss,md}\"",
     "check": "yarn run format:check && yarn run lint",
     "fix": "yarn run format && yarn run lint:fix"
   },
+  "resolutions": {
+    "@types/react": "18.2.8"
```

Comment:
This will throw a warning : `warning Resolution field "@types/react@18.2.8" is incompatible with requested version "@types/react@18.0.32" `

This is probably because we had to pin the version of `types/react` to 18.0.32 : https://github.com/NASA-IMPACT/veda-ui/pull/976#issuecomment-2196111216 I am honestly not sure what to look out 🤔 (Without updated version of types, Promise<Servercomponent> won't work